backend/clickhouse/events.go

```diff
@@ -161,12 +166,17 @@ func (client *Client) EventsKeys(ctx context.Context, projectID int, startDate t
 	return eventKeys, nil
 }
 
-func (client *Client) EventsKeyValues(ctx context.Context, projectID int, keyName string, startDate time.Time, endDate time.Time, query *string, limit *int) ([]string, error) {
+func (client *Client) EventsKeyValues(ctx context.Context, projectID int, keyName string, startDate time.Time, endDate time.Time, query *string, limit *int, event *string) ([]string, error) {
 	if eventBooleanKeys[keyName] {
 		return []string{"true", "false"}, nil
 	}
 
-	return KeyValuesAggregated(ctx, client, EventKeyValuesTable, projectID, keyName, startDate, endDate, query, limit)
+	whereClauses := []string{}
+	if event != nil && *event != "" {
+		whereClauses = append(whereClauses, fmt.Sprintf("Event = %s OR Event = ''", *event))
```

Comment:
just checking that this is not vulnerable to sql injection since the `event` is unchecked user input, but i think the `whereClauses` in `KeyValuesAggregated` is checked by `sb.Var`

Reply:
Good catch! Updated to pass in the event into `KeysAggregated` and handle it in there with `sb.Var`